src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -1359,6 +1359,20 @@
                                    reports/resources->resource-events
                                    (map normalize-resource-event)))))
 
+(defn filter-filter
+  [timestamp resource-events-ttl]
+  (let [after (time/after? (org.joda.time.DateTime. timestamp)
```

Comment:
Don't need the let block here, you can call the `time/after?` func directly and its result will be returned.

Reply:
Updated, forgot to remove them after debugging.